src/borg/repository.py

```diff
@@ -138,6 +139,7 @@ def __enter__(self):
         if self.do_create:
             self.do_create = False
             self.create(self.path)
+            self.created = True
```

Comment:
doesn't self.do_create already reflect the same state information?
or does it need to be False while self.create() is executed?